core/domain/feedback_services.py

```diff
@@ -92,7 +93,7 @@ def create_thread(
 
 def create_message(
         exploration_id, thread_id, author_id, updated_status, updated_subject,
-        text):
+        text, received_via_email=False):
```

Comment:
Here and elsewhere: make sure you update any docstrings that need to be updated.

(Though something seems a bit off in the docstrings for this function; the args don't seem to match. Please could you fix it?)